### PR TITLE
fix(various posts): Fix typos

### PR DESCRIPTION
**Pull request by ericcarraway**
Your blog is great, Todd!

I'm experimenting with a few tools to automate typo-hunting.  I hope you find these corrections helpful.

-Eric

### CODE REVIEW

File: _posts/2013-03-01-progressively-enhanced-svg-sprite-icons.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ Here's an example of two icons:
...
+The image downscaling technique, a good idea at the time but with terrible output. I say this because of a few things. Creating x2 images is such a bad solution to overcoming retina displays, then downscaling the image to achieve a good 'retina' image. It's a drag on performance, a cheap technique which was popularised at the time, but SVG is the real future of website graphics, creating infinitely scalable graphics that are super-futureproof. Image downscaling is not future proof at all, and when a proper solution comes along, previous techniques will be obsolete and not even part of any standard.
```
Comment (ericcarraway) [inline _posts/2013-03-01-progressively-enhanced-svg-sprite-icons.md]: `obselete` =>  `obsolete`
doesn't show up well on my view of GitHub's diff

File: _posts/2013-06-01-reengineering-the-jbar-plugin-with-immediately-invoked-data-expressions.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ Here's the old plugin configuration for the jBar version 1.0.0:
...
 
 ### No more default options
-One thing you're probably used to seeing is jQuery plugin _defaults_ that are actually inside the plugin script itself which you essentially override with your own _options_ when you call the script. Using the IIDE methodology, I decided this wasn't at all necessary and completely ignored the need for it. Why? Because it's not really needed either - the defaults are in the markup for one, and secondly, we can check to see if the user has added the JSON data to the data-&#42; attribute and run a function if necesary - light work.
+One thing you're probably used to seeing is jQuery plugin _defaults_ that are actually inside the plugin script itself which you essentially override with your own _options_ when you call the script. Using the IIDE methodology, I decided this wasn't at all necessary and completely ignored the need for it. Why? Because it's not really needed either - the defaults are in the markup for one, and secondly, we can check to see if the user has added the JSON data to the data-&#42; attribute and run a function if necessary - light work.
```
Comment (ericcarraway) [inline _posts/2013-06-01-reengineering-the-jbar-plugin-with-immediately-invoked-data-expressions.md]: `necesary ` =>  `necessary `
doesn't show up well on my view of GitHub's diff

File: _posts/2014-01-29-mastering-the-module-pattern.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ We then have `Module` declared in the global scope, which means we can call it w
...
+What _are_ private methods you might be asking? Private methods are anything you don't want users/devs/hackers to be able to see/call outside the scope they're in. We might be making server calls and posting sensitive data, we _don't_ want to expose those functions publicly, they could post anything back then and take advantage of our code. So we can create closure and be more sensible (as best as we can with JavaScript) at protecting our code. It's not _all_ about protection however, there are also naming conflicts. I bet when you first started out writing jQuery/JavaScript, that you dumped all your code in one file and it was just `function, function, function`. Little did you know these were all global, and you probably suffered the consequence at some point. If so, you'll learn why, and what to do to change it.
```
Comment (ericcarraway) [inline _posts/2014-01-29-mastering-the-module-pattern.md]: `publically ` =>  `publicly `
doesn't show up well on my view of GitHub's diff

**Comment by toddmotto** [comment]
Oh wow, I think I need your tool hahaha! Thanks so much.

**Pull request merged**